whois-api/src/main/java/net/ripe/db/whois/api/mail/dequeue/MessageParser.java

```diff
@@ -69,7 +70,7 @@ public MailMessage parse(final MimeMessage message, final UpdateContext updateCo
         if (deliveryDate != null && deliveryDate.length > 0 && deliveryDate[0].length() > 0) {
             messageBuilder.date(deliveryDate[0]);
```

Comment:
when no delivery date is found in `MimeMessage` message, `now` in `utc` is set for the date. Confirm in the case where there is a delivery date in  `MimeMessage` message that the timezone is aligned with when `now` is used

Reply:
Do we need an integration test for this scenario?

Reply:
The code has been updated to reflect the latest version.

But having said that not sure how we can have this covered by integration test as it depends on checking that whatever timestamp the smtp server uses, is in UTC. And this can only be confirmed by checking directly on the servers.

Reply:
Can't we set ```Delivery-date:``` to whatever we want in a test email for an integration test?

Reply:
Although not an integration test, I believe this [here](https://github.com/RIPE-NCC/whois/blob/325e925b393584725073828409b4aa8fb79f346b/whois-api/src/test/java/net/ripe/db/whois/api/mail/dequeue/MessageParserTest.java#L96) covers that part of the code

Reply:
Perfect